Src/AutoFixture/Kernel/PropertySpecification.cs

```diff
@@ -95,7 +95,9 @@ private bool PropertyIsCompatibleWithTargetType(object request)
 
         private bool PropertyHasTargetName(object request)
         {
-            return ((PropertyInfo)request).Name == this.targetName;
+            return ((PropertyInfo)request)
+                   .Name
+                   .Equals(this.targetName, StringComparison.OrdinalIgnoreCase);
```

Comment:
See [this comment](https://github.com/AutoFixture/AutoFixture/pull/318#discussion_r26196130).